packages/OS400/chkstrings.c

```diff
@@ -34,7 +34,7 @@
  * made, the EXPECTED_STRING_LASTZEROTERMINATED/EXPECTED_STRING_LAST
  * values can be updated to match the latest enum values in urldata.h.
  */
-#define EXPECTED_STRING_LASTZEROTERMINATED  (STRING_TEMP_URL + 1)
+#define EXPECTED_STRING_LASTZEROTERMINATED  (STRING_DNS_LOCAL_IP6 + 1)
```

Comment:
Why not use `STRING_LASTZEROTERMINATED` for this?

Reply:
It's purposely like that to check for runtime inconsistencies I think.

/cc @jonrumsey

Reply:
Yes, the #define in chkstrings.c is expected to be be manually updated whenever STRING_LASTZEROTERMINATED is updated. The value is compared as part of the OS/400 build process to detect a potential missing code change to the EBCDIC<->ASCII string conversion in curl_easy_setopt_ccsid().